packages/cloud_firestore/cloud_firestore_web/lib/cloud_firestore_web.dart

```diff
@@ -129,26 +129,28 @@ class FirebaseFirestoreWeb extends FirebaseFirestorePlatform {
 
   @override
   set settings(Settings settings) {
-    int? cacheSizeBytes;
-    if (settings.cacheSizeBytes == null) {
-      cacheSizeBytes = 40000000;
-    } else if (settings.cacheSizeBytes == Settings.CACHE_SIZE_UNLIMITED) {
-      // https://github.com/firebase/firebase-js-sdk/blob/e67affba53a53d28492587b2f60521a00166db60/packages/firestore/src/local/lru_garbage_collector.ts#L175
-      cacheSizeBytes = -1;
+    // Union type MemoryLocalCache | PersistentLocalCache
+    dynamic localCache;
+    final persistenceEnabled = settings.persistenceEnabled;
+    if (persistenceEnabled == null || persistenceEnabled == false) {
+      localCache = firestore_interop.memoryLocalCache(null);
     } else {
-      cacheSizeBytes = settings.cacheSizeBytes;
+      localCache = firestore_interop
```

Comment:
After deprecation of `enablePersistence` we no longer have an API to specify multiple tab behavior on Web. So, which tab manager will be used by default when I set `persistenceEnabled: true` on Web?